airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -523,25 +523,25 @@ private JobInfoRead createNewSchedulerCancellation(final Long id) throws IOExcep
   }
 
   private JobInfoRead createManualRun(final UUID connectionId) throws IOException {
-    final ManualSyncSubmissionResult manualSyncSubmissionResult = eventRunner.startNewManualSync(connectionId);
+    final ManualOperationResult manualOperationResult = eventRunner.startNewManualSync(connectionId);
 
-    if (manualSyncSubmissionResult.getFailingReason().isPresent()) {
-      throw new IllegalStateException(manualSyncSubmissionResult.getFailingReason().get());
+    if (manualOperationResult.getFailingReason().isPresent()) {
+      throw new IllegalStateException(manualOperationResult.getFailingReason().get());
     }
 
-    final Job job = jobPersistence.getJob(manualSyncSubmissionResult.getJobId().get());
+    final Job job = jobPersistence.getJob(manualOperationResult.getJobId().get());
 
     return jobConverter.getJobInfoRead(job);
```

Comment:
I think this same few lines of code is repeated 3 times. Now that all of the event runners are returning the same `ManualOperationResult`, I think it would be worthwhile to consolidate it down.

Reply:
Done!